src/main/scala/Worksheet.sc

```diff
@@ -0,0 +1,21 @@
+def count(input: Int): List[Int] = {
+  val counter = {
+    (0 until input).toList
+  }
+  counter
```

Comment:
@andrew-armstrong80 Nice solution 👍 Just curious, did you strictly follow a TDD approach for this as notice you've got a Worksheet implemented to try the solutions?
What were your reasons for having a worksheet and TDD? Is there something that you couldn't identify or resolve by writing a test?